satellites/goes16.yaml

```
@@ -0,0 +1,36 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+  latlongrid:
+    description: Gridded products of reflectivity and brightness temperature measured with the Advanced Baseline Imager (ABI) onboard GOES16 (meso-region scans)
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/SATELLITES/GOES-E/{{resolution}}/{{date.strftime("%Y")}}/{{date.strftime("%Y_%m_%d")}}/GOES_{{'%02d' % (channel)}}_8N-18N-62W-50W_{{date.strftime("%Y%m%d")}}.nc
+      auth: null
+      chunks: {}
+    parameters:
+      resolution:
+        description: temporal and spacial resolution of gridded product
+        type: str
+        default: 0.5km_01min
+        allowed: [0.5km_01min]
```

To keep it a little simpler, I'd skip the parameter `resolution` for now, as I currently don't see that it will become necessary. Also if it will become necessary once, we probably want to have a different format. We can add the parameter later as a non-breaking change.

```suggestion
      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/SATELLITES/GOES-E/0.5km_01min/{{date.strftime("%Y")}}/{{date.strftime("%Y_%m_%d")}}/GOES_{{'%02d' % (channel)}}_8N-18N-62W-50W_{{date.strftime("%Y%m%d")}}.nc
      auth: null
      chunks: {}
    parameters:
```

---

I thought I leave it in for now, so that I don't have to make an even longer description string to mention the spatial and temporal resolution, rather let the resolution parameter speak for itself.

---

@d70-t you agree with me here?!

---

No, as adding the `resolution` parameter creates an additional restriction for the future evolution of the catalog. As soon as we accept that parameter into the catalog, we have to assume that people are using it, even if there is only one option. If for some reason, we find out that this parameter was not a good choice, it will be hard to get rid of it.

If we omit the parameter now, and we find out later that we really need it, we can add it at any time in the future without creating an incompatible change for the uses.

But I wouldn't be too pedantic in this case and would accept the PR even with that parameter in place.